Segfault trying to run config validate.

- [x] I have read [Contribution Guidelines](https://github.com/CircleCI-Public/circleci-cli/blob/master/CONTRIBUTING.md).
- [x] I have checked for similar issues and haven't found anything relevant.
- [x] This is not a security issue (which should be reported here: https://circleci.com/security/)

**Do you want to request a *feature* or report a *bug*?**
Bug
**What is the current behavior?**
Segfault when trying to validate my config. 

Can you provide an example?


```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x7000e1]

goroutine 1 [running]:
github.com/CircleCI-Public/circleci-cli/pipeline.FabricatedValues(0xc0000f3d78)
        /home/circleci/project/pipeline/pipeline.go:20 +0x61
github.com/CircleCI-Public/circleci-cli/cmd.validateConfig(0xc000531f80, 0xc000417cc0, 0x10a4918, 0x0, 0x0, 0x28, 0x14)
        /home/circleci/project/cmd/config.go:128 +0x65
github.com/CircleCI-Public/circleci-cli/cmd.newConfigCommand.func4(0xc00039e000, 0x10a4918, 0x0, 0x0, 0x0, 0x0)
        /home/circleci/project/cmd/config.go:65 +0x5d
github.com/spf13/cobra.(*Command).execute(0xc00039e000, 0x10a4918, 0x0, 0x0, 0xc00039e000, 0x10a4918)
        /go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:762 +0x465
github.com/spf13/cobra.(*Command).ExecuteC(0xc0001a5b80, 0x0, 0x0, 0x9887f7)
        /go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:852 +0x2ec
github.com/spf13/cobra.(*Command).Execute(...)
        /go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:800
github.com/CircleCI-Public/circleci-cli/cmd.Execute()
        /home/circleci/project/cmd/root.go:34 +0x28
main.main()
        /home/circleci/project/main.go:9 +0x20
```


**What is the expected behavior?**
No segfault
**Which version of the CLI and OS are you using? Did this work in previous versions?**
Worked this weekend. Currently using Ubuntu 20.04, with Snap installation version 0.1.7487
Please provide the output of `circleci version` and `circleci diagnostic`.
Version:
`0.1.7487+08d04b0 (snap)`

Diagnostic:
```

---
CircleCI CLI Diagnostics
---
Debugger mode: false
Config found: /home/brady/snap/circleci/166/.circleci/cli.yml
API host: https://circleci.com
API endpoint: graphql-unstable
OK, got a token.
Trying an introspection query on API... 
Ok.
Hello, Brady Sullivan.
```

**If you have any questions, feel free to ping us at @CircleCI-Public/x-team.**

--- a/CONTRIBUTORS.md
+++ b/CONTRIBUTORS.md
@@ -15,3 +15,4 @@ We are very grateful to the following people have helped us to build the CLI too
 - [Ed Serzo](https://github.com/eserzomcity)
 - [Daniel Ruthardt](https://github.com/DanielRuthardt)
 - [Josef Šimánek](https://github.com/simi)
+- [Brady Sullivan](https://github.com/d1str0)

--- a/pipeline/pipeline.go
+++ b/pipeline/pipeline.go
@@ -16,7 +16,8 @@ func FabricatedValues() Values {
 	gitUrl := "https://github.com/CircleCI-Public/circleci-cli"
 	projectType := "github"
 
-	if remote, err := git.InferProjectFromGitRemotes(); err != nil {
+	// If we encounter an error infering project, skip this and use defaults.
+	if remote, err := git.InferProjectFromGitRemotes(); err == nil {
 		switch remote.VcsType {
 		case git.GitHub:
 			gitUrl = fmt.Sprintf("https://github.com/%s/%s", remote.Organization, remote.Project)

